### PR TITLE
Automated cherry pick of #1747: Run make gen-versions to update CRD

### PR DESCRIPTION
**Pull request by doublek**
Cherry pick of #1747 on release-v1.25.

#1747: Run make gen-versions to update CRD

### CODE REVIEW

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
